### PR TITLE
added analytics to events on button and route change

### PR DESCRIPTION
**Pull request by wintersummermint**
### All Submissions

* [x] Can you post a screenshot of your changes (if applicable)?
zS2Jh0Lv3O

### New Feature Submissions

* [x] Does this relate to an existing issue, or has it been talked about prior to being done (if a major change)?

### Notes

Please note, your code must pass all tests and lint checks before it can be merged.

### PR Description

(insert description here)

### CODE REVIEW

File: frontend/src/App.vue [hunk 1]
```diff
@@ -141,6 +150,12 @@ export default {
     this.checkStorage();
 
     Events.$on('setting:rewards', () => this.checkStorage());
+
+    document.body.addEventListener('click', (e) => {
+      if(e.target.nodeName==='BUTTON') {
+        window.gtag('event', 'button_clicked', {method: 'Google'});
```
[the remark this answers is not in the document]
Comment (wintersummermint) [reply frontend/src/App.vue:156]: Screen Shot 2021-06-28 at 9 16 12 AM
Hey, would like to know what should i track is it class name or other else? we might add new attribute?

[the remark this answers is not in the document]
Comment (seiyria) [reply frontend/src/App.vue:156]: We might have to track a custom attribute, but the idea is to track whether people click Forge, Recruit, etc. So we want to track each button based on type. In the case of Recruit and Forge, we want to track if they clicked the button in the inventory, or if they clicked the "first time" recruit or forge button separately.

[the remark this answers is not in the document]
Comment (wintersummermint) [reply frontend/src/App.vue:156]: Ok, im moving forward with adding attribute to each, so i can automatically gather info when it needs to be sent

**Comment by seiyria** [comment]
closing in favor if #133

**Pull request closed without merging**